### PR TITLE
Setup the Project

### PR DESCRIPTION
**Pull request by petushka1**
# Milestone 1
## Creating a data model

- [x] Create ROR blog app
- [x] Initialize with git
- [x] Push to remote
- [x]  install rubocop
- [x] add linters configuration files
- [x] Create a database following the diagram
- [x] set up foreign keys
- [x] create index for each foreign key

### Requirement
:octocat: Correct Gitflow
:heavy_check_mark: No Linters errors
:heavy_check_mark: Descriptive README
:heavy_check_mark: KISS, DRY, YAGNI

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -1,25 +1,7 @@
...
-
-production:
-  <<: *default
-  database: db/production.sqlite3
+  adapter: postgresql
+  encoding: unicode
+  database: projectname
+  pool:
+  username: rails
+  password: rails
```
Comment (danmainah) [inline config/database.yml:49]: - [ ] - [ ] Kindy reset your PostgreSQL configuration, 
```suggestion
  ```ruby
default: &default
adapter: postgresql
encoding: unicode
```
```
Also set environment for development test and production

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @petushka1 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @petushka1, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- The PR title and description looks professional✔️
- No linter errors✔️

 ## Required Changes ♻️
_Check the comments under the review._

- [ ] Let's make the changes requested by the previous reviewer, it's important that we know how to setup the database in a Rails application✔️✔️

## Optional suggestions

- [ ] The readme file doesn't match the current project description. Kindly take a look at it.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by petushka1** [comment]
re-set database.yml

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**